front/src/views/Boxes/BoxesView.tsx

```diff
@@ -215,6 +233,20 @@ function Boxes() {
         disableFilters: true,
         sortType: "datetime",
       },
+      {
+        Header: "Last Modified By",
+        accessor: "lastModifiedBy",
+        id: "lastModifiedBy",
+        Filter: SelectColumnFilter,
+        filter: "includesOneOfMulipleStrings",
+      },
+      {
+        Header: "Created By",
+        accessor: "createdBy",
+        id: "createdBy",
+        Filter: SelectColumnFilter,
+        filter: "includesOneOfMulipleStrings",
```

Comment:
didn't originate from this PR, could you still fix the spelling though (missing t in "Multiple")?